csfieldguide/templates/interactives/angle-of-view-calculator.html

```diff
@@ -0,0 +1,53 @@
+{% extends interactive_mode_template %}
+
+{% load i18n %}
+{% load static %}
+
+{% block html %}
+<div id="angle-of-view-calculator">
+  <h5>{% trans "Calculate the angle of view" %}</h5>
+
+  <div class="input-group mb-3">
+    <div class="input-group-prepend">
+      <span class="input-group-text" id="ruler-label">Ruler length visible (r)</span>
+    </div>
+    <input id="ruler-length" type="number" step="0.01" class="form-control" autocomplete=off aria-label="ruler-length" aria-describedby="ruler-label">
+    <div class="input-group-append">
+      <button class="btn btn-outline-secondary dropdown-toggle" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" id="ruler-unit">Metres</button>
+      <div class="dropdown-menu" id="ruler-unit-list">
+        <a class="dropdown-item" href="#">Metres</a>
+        <a class="dropdown-item" href="#">Inches</a>
+      </div>
+    </div>
+  </div>
+
+  <div class="input-group mb-3">
+    <div class="input-group-prepend">
+      <span class="input-group-text" id="distance-label">Distance to ruler (d)</span>
+    </div>
+    <input id="ruler-distance" type="number" step="0.01" class="form-control" autocomplete=off aria-label="ruler-length" aria-describedby="distance-label">
+    <div class="input-group-append">
+      <button class="btn btn-outline-secondary dropdown-toggle" type="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" id="distance-unit">Metres</button>
+      <div class="dropdown-menu" id="distance-unit-list">
+        <a class="dropdown-item" href="#">Metres</a>
+        <a class="dropdown-item" href="#">Inches</a>
+      </div>
+    </div>
+  </div>
+
+  <div id="result-div" class="container">
+    <button id="go-button" class="btn btn-outline-primary">Go!</button>
```

Comment:
Would recommend changing the label to inform the user what is being done. You will want to change any references in the text, plus you may want to change ID of element as well.

```suggestion
    <button id="go-button" class="btn btn-outline-primary">Calculate angle of view</button>
```

Reply:
Renamed wording and id.